src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -277,38 +282,31 @@ class DemoPage extends StatelessWidget {
 }
 ```
 
-1. Run the app. When you click the 'Show Flutter homepage' you should see the
-   phone's default browser open, and the Flutter homepage appear.
+1. Run the app. The app's background should now be orange.
```

Comment:
In the rendered document, this is back to #1 but should be #5. I think the previous code snippet needs to be indented.